src/components/SelectPlayStep.tsx

```diff
@@ -24,7 +24,7 @@ export default class SelectPlayStep extends React.PureComponent<Props> {
           {Object.keys(Play).filter(key => !isNaN(Number(key))).map(option => (
             <button
               type="button"
-              onClick={() => chooseAPlay(Play[option])}
+              onClick={() => chooseAPlay(parseInt(option,10))}
```

Comment:
The enum was not parsing properly before, so the compare logic would not work correctly. Probably a better way to do this.